### ISSUE TITLE
Feature request: Auto enable all coins with balance above a preset fiat value

### ISSUE THREAD

**Issue by gcharang** [KomodoPlatform/atomicDEX#138]
Allow users to set a minimum fiat value in the settings. If set, the app should query an API(maybe dexstats.info or other explorers) for balances of all supported coins based on the pubkey and then multiply the balance with the respective fiat prices. If the fiat value of the balance of a coin exceeds the value set in the settings, the coin should be enabled and displayed in the portfolio tab.

**Comment by dth88** [comment]
@gcharang can you elaborate here please, why is that needed? It would help if you give some context. Did the app feel slow with many coins activated and this is the way you propose to make it feel faster by getting rid of 0-balance coins?

**Comment by gcharang** [comment]
context: https://discord.com/channels/412898016371015680/584762059430821909/682943946418028574
posting the message here:
> Dochteur: Is it normal for the app to remove all the wallets from the list apart from BTC and KDM, when it is reinstalled? I always have to set them back afterwards. Is there a possibility that this could change the adresses, if I had any balance there, making these coins unreachable? (with the exception of BTC and KMD assetchains, which use the same address, so for example LTC, ETH and others)

this is so that users will know all the coins that they own above a certain value. right now there is no easy way to know which coins even have balances. to know balances, the coins need to be first enabled in mm2, and if some don't have any balance, they have to be disabled manually again. all of this is very time consuming. when enabled, the app's portfolio tab behaves like a CEX's balances page that lists all the coins the user owns 

this feature basically automates what any user does after a reinstall and is also useful for users returning after a long time.

**Comment by tonymorony** [comment]
yeah, thats the real problem. I do not remember 100% of coins I got balance for too - and amount of coins will only grow in future. 
Maybe as a resolution it's fine do display coin balances on this screen and/or have button like a "Activate all coins with >0 balances" there? 🤔 (just possible alternatives to detection/activation on seed restore stage)

![image](https://user-images.githubusercontent.com/24797699/83356520-f5a3eb00-a390-11ea-9f71-a7d11b7307ef.png)
